source/reference/minio-mc/mc-ready.rst

```diff
@@ -0,0 +1,112 @@
+============
+``mc ready``
+============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc ready
+
+Syntax
+------
+
+.. start-mc-ready-desc
+
+The :mc:`mc ready` command checks the status of a cluster and whether the cluster has ``read`` and ``write`` quorum.
+
+.. end-mc-ready-desc
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following sends a ``GET`` request to the cluster and returns its status.
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc ready play
+
+      The command sends a ``GET`` request to the deployment at the :mc:`~mc alias` ``play``.'
+      The command repeats the request until it is successful.
+
+      The output before a cluster is ready resembles the following:
+
+      .. code-block:: shell
+
+         The cluster is unreachable: Get "http://play.min.io:9000/minio/health/cluster": dial tcp 127.0.0.1:9000: connect: connection refused
+
+      Once the request succeeds, the output resembles the following:
+     
+      .. code-block:: shell
+
+         The cluster is ready
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc [GLOBALFLAGS] ready            \
+                          TARGET           \
+                          [--cluster-read] \
+                          [--maintenance]
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: TARGET
+   :required:
+
+   The full path to the :ref:`alias <minio-mc-alias>` or prefix where the command should run.
+
+.. mc-cmd:: --cluster-read
+   :optional:
+
+   Checks if the cluster has enough :term:`quorum <read quorum>` to serve ``READ`` requests.
+
+.. mc-cmd:: --maintenance
+   :optional:
+   
+   Checks if the cluster can maintain read and write quorum if taken down for maintenance.
```

Comment:
if what is taken down? the cluster?

I assume the *node* that receives the ready command - if a load balancer is involved then it will be hard for the user to know what node is safe to take down.

I wonder if we need guidance to use this with a custom alias for the specific node you want to pop.